SCClassLibrary/deprecated/3.10/deprecated-3.10.sc

```diff
@@ -97,3 +97,17 @@ SharedIn : AbstractIn {
 		this.deprecated(thisMethod, this.class.findRespondingMethodFor(\run))
 	}
 }
+
++ AbstractFunction {
+	asInt {
+		this.deprecated(thisMethod, this.class.findMethod(\asInteger));
+		^this.composeUnaryOp('asInteger');
+	}
+}
+
++ Object {
+	asInt {
+		this.deprecated(thisMethod, this.class.findMethod(\asInteger));
+		^this.asInteger;
+	}
+}
```

Comment:
@snappizz should these deprecations go into a new file called `deprecated-3.11.sc`?

Reply:
yes, this should be moved to `SCClassLibrary/deprecated/3.11/deprecated-3.11.sc`

Reply:
sorry @davidgranstrom , i missed that during review!

Reply:
@brianlheim No problem! I updated the last commit to reflect this (also rebased).